.gitignore

```diff
@@ -2,3 +2,4 @@
 **/*.jar
 **/*.zip
 **/*.tar.gz
+.idea/
```

Comment:
Yeah, I know what `.gitignore` does. :) I meant, what creates the `.idea/` directory and why is it being added to `.gitignore`? This seems like it could be something from a git IDE that's not actually required.